cloud/clouds.go

```diff
@@ -139,6 +142,7 @@ type cloudSet struct {
 // cloud is equivalent to Cloud, for marshalling and unmarshalling.
 type cloud struct {
 	Type             string                 `yaml:"type"`
```

Comment:
Why don't we have matching json tags here? We should have some that match the yaml tags.